packages/core/state/interaction/reducer.ts

```diff
@@ -50,9 +51,9 @@ export interface InteractionStateBranch {
     csvColumns?: string[];
     dataSourceInfoForVisibleModal?: DataSourcePromptInfo;
     datasetDetailsPanelIsVisible: boolean;
-    fileExplorerServiceBaseUrl: string;
     fileTypeForVisibleModal: "csv" | "json" | "parquet";
     fileFiltersForVisibleModal: FileFilter[];
+    environment: "LOCALHOST" | "STAGING" | "PRODUCTION" | "TEST";
```

Comment:
same alphabetical ordering nit throughout file

Reply:
resolved in [b482dcf](https://github.com/AllenInstitute/biofile-finder/pull/296/commits/b482dcfc948a9bdd213afcb0021f679ba1bb6da8)